app/models/link_history.rb

```diff
@@ -0,0 +1,24 @@
+class LinkHistory < ApplicationRecord
+  belongs_to :link
+
+  def add_errors(errors)
```

Comment:
Do we need to think about the warnings as well as the errors?

Reply:
At the moment we are just focusing on the errors, but its in our backlog, so will look into it then

Reply:
Hmm, okay. I wonder if it's worth putting a comment in to explain this as I'm a bit worried that feature will be forgotten.